test/jdk/java/awt/Graphics/LCDTextAndGraphicsState.java

```diff
@@ -21,56 +21,84 @@
  * questions.
  */
 
-/**
+/*
  * @test
  * @bug 6576507
  * @summary Both lines of text should be readable
- * @run main/manual=yesno LCDTextAndGraphicsState
+ * @run main/manual LCDTextAndGraphicsState
  */
-import java.awt.*;
-import java.awt.geom.*;
+
+import java.awt.BorderLayout;
+import java.awt.Button;
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.Dimension;
+import java.awt.Frame;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.GridBagLayout;
+import java.awt.GridBagConstraints;
+import java.awt.RenderingHints;
+import java.awt.AlphaComposite;
+import java.awt.GradientPaint;
+import java.awt.TextArea;
+import java.awt.Shape;
+import java.awt.Panel;
+import java.awt.Insets;
+
+import java.awt.event.ActionEvent;
+import java.awt.geom.RoundRectangle2D;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
 
 public class LCDTextAndGraphicsState extends Component {
 
-    String text = "This test passes only if this text appears SIX TIMES";
+    private static final Frame instructionFrame = new Frame();
+    private static volatile boolean testResult = false;
+    private static volatile CountDownLatch countDownLatch;
```

Comment:
It could be `final` and could be initialised here rather than in main.

Reply:
Both final & volatile can be used it gives error "Illegal combination of modifiers: volatile & final"

Reply:
> Both final & volatile can be used it gives error "Illegal combination of modifiers: volatile & final"

That's right: both final and volatile don't make sense.

What I meant was
```suggestion
    private static final CountDownLatch countDownLatch = new CountDownLatch(1);
```